MachObfuscatorTests/OptionsTests/Options_Tests.swift

```diff
@@ -1,11 +1,13 @@
 import XCTest
 
-class Options_Tests: XCTestCase {
+class OptionsTestsSupport: XCTestCase {
```

Comment:
What do you think about extracting this to other file? Up to you.

Reply:
I was thinking about it too :) Done.